docs/using-the-compiler.rst

```diff
@@ -178,6 +178,7 @@ at each version. Backward compatibility is not guaranteed between each version.
    - Smaller code size and gas savings due to the introduction of ``push0`` (see `EIP-3855 <https://eips.ethereum.org/EIPS/eip-3855>`_).
 - ``cancun``
    - The block's blob base fee (`EIP-7516 <https://eips.ethereum.org/EIPS/eip-7516>`_ and `EIP-4844 <https://eips.ethereum.org/EIPS/eip-4844>`_) can be accessed via the global ``block.blobbasefee`` or ``blobbasefee()`` in inline assembly.
+   - Introduces ``blobhash()`` in inline assembly and a corresponding global function to retrieve versioned hashes of blobs associated with the transaction (see `EIP-4844 <https://eips.ethereum.org/EIPS/eip-4844>`_).
```

Comment:
```suggestion
   - Introduces ``blobhash(uint index)`` in inline assembly and a corresponding global function to retrieve versioned hashes of blobs associated with the transaction (see `EIP-4844 <https://eips.ethereum.org/EIPS/eip-4844>`_).
```

Reply:
There's no `uint` in Yul :)

Reply:
> Huh, is inline assembly not Yul?

It is and there's no `uint` in inline assembly either. The opcodes are untyped. You can access `uint` variables defined at Solidity level but they're seen as just bytes.

`blobhash(index)` would be acceptable, but not sure we really need that detail here.